adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
The same applies to iOs
I know Jamie had the same #ifdef but with

#if TARGET_OS_IPHONE

If you add both of them here we can also close Jamie's PR.